src/web/app/pages-instructor/instructor-sessions-page/instructor-sessions-page.component.ts

```diff
@@ -124,8 +135,6 @@ export class InstructorSessionsPageComponent extends InstructorSessionModalPageC
   isSessionEditFormExpanded: boolean = false;
 
   sessionsTableRowModels: SessionsTableRowModel[] = [];
-  sessionsTableRowModelsSortBy: SortBy = SortBy.NONE;
-  sessionsTableRowModelsSortOrder: SortOrder = SortOrder.ASC;
```

Comment:
No longer necessary to track the sorting state, since the `sortable-table` component now emits the sorting state. 
See [comment](https://github.com/TEAMMATES/teammates/pull/12501/files#r1254558184).